Marlin/src/gcode/feature/pause/M125.cpp

```diff
@@ -87,14 +87,20 @@ void GcodeSuite::M125() {
   #endif
 
   if (pause_print(retract, z_lift, x_pos, y_pos)) {
+    //disabling Run Next Tool feature 
+	  #if ENABLED(PAUSE_SPOOL_SWAP)
+	    bool swap_spool_disabling = swap_spool_enabled;//save momentary
+	    swap_spool_enabled = false;//Runout Next Tool OFF
+    #endif 	  
```

Comment:
m125 is  just pause , and in M125 swap spool never exist 
it's why , i disable swapspool status  , and recover when finish

Reply:
i don't use M125 but i know many user love it , to change filament and make folkloric multicolor print
its why i have securised this code